lib/x11utils.pm

```diff
@@ -92,6 +92,7 @@ sub ensure_unlocked_desktop {
         }
         wait_still_screen 2;       # slow down loop
         die 'ensure_unlocked_desktop repeated too much. Check for X-server crash.' if ($counter eq 1);    # die loop when generic-desktop not matched
+        record_info('Desktop unlock attempt failed', "Try #$counter failed", result => 'softfail');       #mark attempts
```

Comment:
I consider this too noisy. The "softfail" and the mention of "failed" will most likely confuse many people in too many scenarios as there are flows where multiple loop cycles are expected and this is nothing wrong.
Also, did you actually see the unlock failing with the die message just above? E.g. in https://openqa.opensuse.org/tests/854596#step/updates_packagekit_gpk/1 one can see the black screen which is detected as "screenlock" so we press "esc" and IMHO that is correct but we still need to check for the next screen which I would not label as "softfail" with a message that an unlock attempt "failed" because we just pressed "esc", we did not even try to unlock. It's like "wiggling your mouse" to wait for the actual unlock screen.

How about just writing "diag" here instead of "record_info", and not "failed" but just the information about the counter value?

Reply:
@okurz So Let's then change the soft fail to a no failure type at all, so that people won't get confused. The method is called "ensure unlocked desktop", so what would you prefer to write? `I tried to unlock this, but I need another try`. 

About the noise, I see more noisy (Actually I call it waste of time from the perspective of a reviewer) to have to go through the logs, trying to figure out how many times did the code actually loop through, increasing the time required for a review

Reply:
> I tried to unlock this, but I need another try.

But that's the thing. It does not even try to unlock when it just encounters the screensaver.

Reply:
@okurz I accept suggestions for messages then ;)

Reply:
how about `record_info('desktop unlock', 'count: $counter')`

But please provide at least one job in openQA so that we can see how that looks and if it's helpful. Also I am never sure about the line wrapping in `record_info` so we should actually see it first.

Reply:
Sure thing :), waiting for: http://phobos.suse.de/tests/1747267 to finish

Reply:
http://phobos.suse.de/tests/1747268#step/brasero_launch/14 shows it better

Reply:
Speaking of the devil: https://openqa.suse.de/tests/2478886#step/yast2_lan_restart_devices/20